documentation/developing/modding_examples/blindfold_assist.mdx

```diff
@@ -0,0 +1,175 @@
+---
+sidebar_position: 0
+title: Blindfold Assist
+description: A deep dive into the code changes for the Blindfold Assist Mod
+---
+
+# Blindfold Assist Mod - Code Review
+February 16, 2023
+<img style={{verticalAlign:"middle"}} src="https://github.com/dallmeyer.png" width="40px"/> <a href="https://github.com/dallmeyer"><b>barg034</b></a>
```

Comment:
probably a better way to so this? I tried to mimic the headings from the Progress Reports, but using `authors` seemed like it was only configured to work for the blog pages?

![image](https://user-images.githubusercontent.com/2515356/219613524-877f34e7-6a9e-450d-9a9e-6bff69aeff35.png)